### ISSUE TITLE
Typo on reduction clause

### ISSUE THREAD

**Issue by chichunchen** [LLNL/RAJA#1365]
https://github.com/LLNL/RAJA/blob/develop/include/RAJA/policy/openmp_target/forall.hpp#L79

Should be `reduction` instead.
```
#pragma omp target teams distribute parallel for num_teams(numteams) \
    schedule(static, 1) map(to : body,begin_it) redcution(combine: f_params)
  for (i = 0; i < distance_it; ++i) {
    Body ib = body;
    RAJA::expt::invoke_body(f_params, ib, begin_it[i]);
  }
```

**Comment by mdavis36** [comment]
PR #1366

**Comment by rhornung67** [comment]
Fixed and merged into develop, included in v2022.10.2 release
